web/src/tippyjs.js

```diff
@@ -448,19 +448,37 @@ export function initialize() {
     });
 
     delegate("body", {
-        target: "#show_all_private_messages",
+        target: [
+            "#show_all_private_messages",
+            ".message_expander",
+            ".message_condenser",
+            ".top_left_all_messages",
+            ".top_left_recent_topics",
+            ".top_left_mentions",
+            ".top_left_starred_messages",
+            ".top_left_drafts",
+            "#settings-dropdown",
+            ".create_stream_button",
+            ".toggle-subscription-tooltip",
+            "#preview-stream-button",
+        ],
         placement: "bottom",
-        content: $t({
-            defaultMessage: "All direct messages (P)",
-        }),
+        onShow(instance) {
+            const $elem = $(instance.reference);
+            const $template = $("#" + $elem.attr("data-tooltip-template-id"));
+            instance.setContent(parse_html($template.html()));
+        },
```

Comment:
What's the reason for removing this and replacing it with a custom function? I'm pretty sure you could just continue using `content` rather than creating a custom function and creating an extra template in the Handlebars file.

Reply:
The onShow function is used instead of content to display hotkey hints in the tooltip. This approach is consistent with that used in PR [#22746](https://github.com/zulip/zulip/pull/22746), which this [issue](https://github.com/zulip/zulip/issues/24311) builds upon. 
If I have misunderstood the issue you raised, please let me know.

Reply:
Sorry, I was the one that misunderstood the issue, that's fine then.